### DISCUSSION TITLE
Few ideas

### DISCUSSION THREAD

**Discussion by privet-fun** [ShufflePerson/Discord_CDN#5] [General]
I was inspired by your approach and added Discord CDN smart proxy to my phpbb extension
https://github.com/privet-fun/phpbb_ailabs?tab=readme-ov-file#discord-cdn-smart-proxy-setup.
In addition to link refresh it also caching results and verify channel(s) as a security measure.

**Comment by ShufflePerson** [comment]
Hey!

The results are already cached, also, what exactly do you mean by verifying channels? There is no difference to allowing or disallowing some channels, as anyone can access the file anyways no matter if they have access to the channel or not. Simply pasting the link in a Discord channel will update the link even if they don't have access to that channel. Doesn't provide any security whatsoever.
